README.md

```diff
@@ -35,7 +35,7 @@ running in the Keptn ecosystem:
 |    0.16.x     |                               keptncontrib/job-executor-service:0.2.3                                |       v2       |
 |    0.16.x     |                               keptncontrib/job-executor-service:0.2.4                                |       v2       |
 |    0.17.x     |                               keptncontrib/job-executor-service:0.2.5                                |       v2       |
-
+|    0.18.x     |                               keptncontrib/job-executor-service:0.2.6                                |       v2       |
```

Comment:
I assume that we are removing version 0.16.1 and 0.17.0 from the test matrix because this release isn't compatible anymore. If so this should also be reflected in the version number and this PR should be marked as a breaking change.

Reply:
You are right. Since we are using the new `resource-service` endpoint, it is no longer compatible with older versions. I have marked the PR as breaking.